src/style/styles.css

```diff
@@ -5,4 +5,48 @@
   padding: 0;
   box-sizing: border-box;
   font-family: 'Poppins', sans-serif;
+  list-style: none;
+  text-decoration: none;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  z-index: 100;
+  background-color: #cfc5c5;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 10% 10px;
+}
+
+header h1 {
```

Comment:
These styles are not applied anywhere because there is no **h1** inside the header.